src/servers/main.cc

```diff
@@ -56,6 +56,10 @@ static_assert(
 #include "src/servers/grpc_server.h"
 #endif  // TRTIS_ENABLE_GRPC
 
+#ifdef TRTIS_ENABLE_KFSERVING
```

Comment:
I don't think we need any changes in this file.

Reply:
Fixed